MIDAS/src/gnc/kalman_filter.h

```diff
@@ -0,0 +1,57 @@
+#ifndef GNC_KALMAN_FILTER_H_
+#define GNC_KALMAN_FILTER_H_
+
+#undef abs
+#undef F
+#undef round
+#undef B1
+
+#include <Eigen/Eigen>
+#include "sensors.h"
+
+template <int _NumStates, int _NumInputs>
+class KalmanFilter
+{
+protected:
+    Eigen::Matrix<float, _NumStates, 1> x_k;
```

Comment:
This isn't specific to your PR, but do we want to start standardizing our data types? Like specifying how many bits we want our ints to be etc. @aidancostello up to you

Reply:
I'm debating between trying to enforce everyone doing it in their individual branches or just having a ticket being ctrl+f replace with standardized types.

Reply:
Even then we might get bit by alignment if we rely on the structs being the same. Using packed structs or forced 4 byte alignment wouldn't be too large of a performance concern, probably.